common/src/chain/mlt.rs

```diff
@@ -30,6 +30,8 @@ impl Mlt {
     pub const ZERO: Self = Self(Amount::from_atoms(0));
     /// Maximum representable amount of MLTs
     pub const MAX: Self = Self(Amount::MAX);
+    /// The coin name
+    pub const NAME: &'static str = "Mlt";
```

Comment:
`TICKER: &'static str = "ML"`

We decided to move away from MLT. I guess it has to be changed in many places.

Reply:
> I guess it has to be changed in many places.

I fixed it here, but let's please do other renames in another PR.

Reply:
Yes that's OK, but you still are calling this "name" in the places you defined.

Reply:
Should be fixed now, thanks.

Reply:
Cheers!